main.cpp

```diff
@@ -92,6 +92,59 @@ good to go!
  */
 
 #include <iostream>
+
+struct FloatType
+{
+public:
+    float add(float lhs, float rhs) { return lhs + rhs; }
+    float subtract(float lhs, float rhs) { return lhs - rhs; }
+    float multiply(float lhs, float rhs) { return lhs * rhs; }
+    float divide(float lhs, float rhs)
+    {
+        if (rhs == 0.0f)
+        {
+            std::cout << "\nwarning, floating point division by zero returns 'inf' !" << std::endl;
+        }
+        return lhs / rhs; 
+    }
+};
+
+struct DoubleType
+{
+public:
```

Comment:
`struct` is `public` by default.
this access specifier is unnecessary
- [ ] revise